js/testapps/basic-gemini/package.json

```diff
@@ -7,20 +7,23 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "node lib/index.js",
     "build": "tsc",
-    "build:watch": "tsc --watch"
+    "build:watch": "tsc --watch",
+    "genkit:dev": "cross-env GENKIT_ENV=dev pnpm dev",
```

Comment:
same thing as in https://github.com/firebase/genkit/pull/1259#discussion_r1844538029